SU-SOE/soe_basic
# D8CORE-2308: changing the logo in the footer

# READY FOR REVIEW

# Summary
- Changed the footer to logo to say Engineering instead of configured site name and to link back to engineering.stanford.edu

# Review By (Date)
- Wednesday

# Urgency
- Med

# Steps to Test

1. Pull in this branch.
2. Clear caches
3. Verify the footer says Engineering and links to the engineering.stanford.edu

# Affected Projects or Products
- SOE Subtheme

# Associated Issues and/or People
- D8CORE-2308

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
